package.json

```diff
@@ -37,7 +37,7 @@
     "prepublish": "tsc",
     "example": "yarn --cwd examples/purchaseTester",
     "pods": "cd examples/purchaseTester && pod-install --quiet",
-    "bootstrap": "yarn examples/purchaseTester && yarn && yarn pods"
+    "bootstrap": "yarn example && yarn && yarn pods"
```

Comment:
This was changed by mistake when renaming the example. `yarn example` is running the script in line 38